### DISCUSSION TITLE
Electrostatic Potential surface

### DISCUSSION THREAD

**Discussion by dulalmondal123** [Electrostatics/apbs#273] [General]
Dear Experts,
I run APBS in VMD. After running APBS, it generates a 'pot. dx' file. In VMD, I select some Hydrophilic residues. Then I calculate electrostatic potential using APBS in VMD. I am trying to calculate the potential value of the selected residue. I add all potential value in pot.dx file to get the electrostatic potential of the selected residue. Is it ok?

Thanking you

Dulal Mondal

**Comment by sobolevnrm** [comment]
Hello --

You should be able to extract the potential value at atom sites with the [write atompot](https://apbs.readthedocs.io/en/latest/using/input/old/elec/write.html) command.  However, these potentials include self-interaction terms which need to be removed by

1. Calculating the potential with a heterogeneous dielectric (e.g., sdie = 80, pdie = 4)
2. Calculating the potential with a homogeneous dielectric (e.g., sdie = 4, pdie = 4)
3. Subtracting the results of Step 2 from Step 1
4. Adding back in the Coulombic potential with a dielectric of 4 (based on the example values above).

Thank you,

Nathan
